### PR TITLE
#3 - adding enums and bumping version

### LINKED ISSUE #3: Consider making "requesting module" an enum
The request objects in the message processor lib have a "requestingModule" field, which seems like it should probably be an `enum`. Since it's shared between MPL and `backend`, and doesn't quite "belong" in the MPL side, `backend-util` would probably be a natural fit.

### PR DESCRIPTION
**Pull request by pveeckhout**
closes #3

### CODE REVIEW

File: src/main/java/io/edpn/backend/util/Topic.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package io.edpn.backend.util;
+
+public interface Topic {
+
+    enum Request {
```
Comment (officialyinsane) [inline src/main/java/io/edpn/backend/util/Topic.java:5]: is "request" the correct naming?

EDDN is not really a request, but an update, in my opinion.

Comment (pveeckhout) [reply src/main/java/io/edpn/backend/util/Topic.java:5]: this is for the inter module communication.

these are the topics where our own modules will put a message to request data from another module

**Review by officialyinsane — APPROVED** [review]

File: src/main/java/io/edpn/backend/util/Topic.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        SYSTEM_COORDINATES_RESPONSE("%s_systemCoordinatesResponse"),
+        SYSTEM_ELITE_ID_RESPONSE("%s_systemEliteIdResponse");
+
+        private final String topicName;
+
+        Response(String topicName) {
+            this.topicName = topicName;
+        }
+
+        public String getFormattedTopicName(String module) {
```
Comment (jakaarl) [inline src/main/java/io/edpn/backend/util/Topic.java]: This could use the enum instead?

**Review by jakaarl — APPROVED** [review]

File: src/main/java/io/edpn/backend/util/Topic.java (hunk 2 above)
Comment (pveeckhout) [reply src/main/java/io/edpn/backend/util/Topic.java]: was thinking the same when coding this....

great minds think alike... I will change it

**Pull request merged**